.github/workflows/infrastructure.yml

```diff
@@ -564,6 +564,26 @@ jobs:
             --name IPAllowListRule \
             --policy-name waf${{ env.PRODUCT }}${{ env.TARGET }}${{ vars.VERSION }}
 
+      - name: Identity 🪪
```

Comment:
emoji not showing for me, or should it be a ?

Reply:
Have updated.